app/views/role_mailer/user_added_to_team.html.slim

```diff
@@ -18,3 +18,5 @@ html
           ' Please confirm that you are indeed willing to join the team as their coach.
           ' You can do so on the team's overview page listed above (make sure you are
           ' logged in using your Github account).
+
+      ' Please visit your profile page, to complete your profile and confirm your email address.
```

Comment:
I'd suggest to straight add a link that the receiver can follow to edit their profile:
```ruby
link_to('your profile page', edit_user_url(@role.user))
```
Maybe make `profile page` in this sentence the link.

Reply:
This URL will only work if the user is already signed in. Should I use it anyway or should I include instead a link to the homepage and ask the user to go to the profile page? Which one do you think is better @klappradla , @F3PiX ?

Reply:
Good point, Luisa. If the user is not logged in, wouldn't the profile link redirect to the login page automatically? Did you check that? Or what happens?

Reply:
In this case I'm redirected to the homepage and I see this message: You are not authorized to access this page.

Reply:
And if you log in after this redirect, does that automatically redirect you to the profile page?

Reply:
Currently, when I log in I see the activities page

Reply:
Yeah, that is not very user-friendly. It should be better if we redirect to the last request. 
IIRC that was discussed before, I'll have to look into that. (And maybe add a new issue.)
Thanks for pointing this out! 

Conclusion for now: (cc @klappradla)
I'd say: put the link to the profile in the email anyway. (so that the user can find the link, if they get lost).